### PR TITLE
TRL-006 Rules_verification

### PR DESCRIPTION
**Pull request by helenver**
[allure.zip](https://github.com/helenver/TRL-Trello/files/11605144/allure.zip) was updated on 30.05 at 23.20
[allure.zip](https://github.com/helenver/TRL-Trello/files/11621673/allure.zip) was updated on 01.06 at 8.30
[allure.zip](https://github.com/helenver/TRL-Trello/files/11627546/allure.zip) was updated on 01.06 at 19.30
[allure.zip](https://github.com/helenver/TRL-Trello/files/11628405/allure.zip) was updated on **01.06 at 21.30**

**Scope:**

_Preconditions:_

- Board creation via API
- Cards creation via API

_Verifications:_

- Trigger creation in the 'Card Move' section
- Action creation in the 'Move' section
- Action creation in the 'Add/Remove' section
- Action creation in the 'Dates' section
- Action creation in the 'Content' section
- Action deletion from the "Created rule" section
- Rule addition
- Rule editing
- Rule "add blue label" implementation
- Rule 'due date as complete' implementation 
- Rule deletion
- Board deletion via API

### CODE REVIEW

**Comment by ruslan-tyshchanka** [comment]
Please make scope more business like. Focus more on identifying the business rules which are verified in this PR, rather then enlisting the steps from test story

File: src/main/resources/story/regression/rules_checks.story [hunk 1]
```diff
@@ -0,0 +1,58 @@
+GivenStories:/story/precondition/create_card_with_details.story
+
+Scenario: Log in as a board creator
+When I log in with ${email} and ${password}
+
+Scenario: Select a trigger in "Card Move" section
```
Comment (ruslan-tyshchanka) [inline src/main/resources/story/regression/rules_checks.story:6]: Comment from PR scope also applies to Scenario names. Story is called rules_checks, and yet there are only two scenario rules with 'rules are saved' and 'rules are deleted'

File: src/main/resources/story/regression/rules_checks.story [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Scenario: Delete an action 'rename the card to'
+When I click on element located by `By.xpath(//h4[text()="Actions"]/../div//div[@class="ui grid item"][4]//i[@class="trash icon"])`
+Then number of elements found by `By.xpath(//textarea[text()="max"])` is equal to `0`
+
+Scenario: Save the created rule 
+When I click on element located by `By.caseSensitiveText(Save)`
+Then number of elements found by `By.xpath(//div[contains(text(),"when a card is added to the board")])` is equal to `1`
+
+Scenario: Remove the created rule
```
Comment (ruslan-tyshchanka) [inline src/main/resources/story/regression/rules_checks.story:50]: The rule is saved, and then immediately deleted in the next scenario. Where are rules verification scenarios?

**Review by ruslan-tyshchanka — CHANGES_REQUESTED** [review]

File: src/main/resources/story/regression/rules_checks.story (hunk 2 above)
Comment (helenver) [reply src/main/resources/story/regression/rules_checks.story:50]: I have added

**Comment by ruslan-tyshchanka** [comment]
Please add rules verification after rule was created and updated, and also check that rule wasn't applied after its deletion

**Review by ruslan-tyshchanka — APPROVED** [review]

**Pull request merged**